src/main/java/LoginTest.java

```diff
@@ -30,11 +33,21 @@ void loginTest() throws InterruptedException {
         }
 
         open(basicURL);
-        $(By.name("loginButton")).shouldBe(Condition.visible);
+        loginPage
+                .getLoginButton()
+                .shouldBe(Condition.visible);
+        loginPage.typeLogin(login);
+        loginPage.typePassword(password);
+        loginPage.clickLoginButton();
+        homePage
+                .getHomepageMarker()
+                .shouldBe(Condition.visible);
+
+        /*$(By.name("loginButton")).shouldBe(Condition.visible);
```

Comment:
please, no commented out code in merge request. merge request always represents finished work